src/index.html

```diff
@@ -11,8 +11,44 @@
       rel="stylesheet"
       href="styles/index.scss"
     />
+    <link
+      rel="stylesheet"
+      href="styles/variables.scss"
+    />
```

Comment:
Same issue here. You have linked a SCSS file. Replace this link with the link to the compiled CSS file.